src/Components/Server/src/DependencyInjection/ComponentServiceCollectionExtensions.cs

```diff
@@ -74,6 +74,7 @@ public static IServerSideBlazorBuilder AddServerSideBlazor(this IServiceCollecti
             services.AddScoped<IJSRuntime, RemoteJSRuntime>();
             services.AddScoped<INavigationInterception, RemoteNavigationInterception>();
             services.AddScoped<AuthenticationStateProvider, ServerAuthenticationStateProvider>();
+            services.AddScoped<IComponentActivator, ComponentActivator>();
```

Comment:
I think this can be now removed since activator will fallback to default implementation anyway.